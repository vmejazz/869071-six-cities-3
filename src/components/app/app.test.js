import React from "react";
import renderer from "react-test-renderer";
import App from "./app.jsx";

const OFFER_PLACES = 254;

const offersArray = [
  {
    id: 1,
    title: `First title`
  }
];

describe(`Test render App component`, () => {

  it(`<App /> should render page`, () => {
    const tree = renderer
      .create(
          <App
            offerPlacesCount={OFFER_PLACES}
            offersArray={offersArray}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
