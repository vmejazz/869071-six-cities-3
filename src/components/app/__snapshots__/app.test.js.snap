// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render App component <App /> should render page 1`] = `<ApartmentDetailInfo />`;
